.github/workflows/coverity.yml

```diff
@@ -10,6 +10,7 @@ on:
   workflow_dispatch:
 jobs:
   coverity:
+    if: github.repository == 'net-snmp/net-snmp'
```

Comment:
Please change this into a test that verifies that both secrets.COVERITY_SCAN_EMAIL and secrets.COVERITY_SCAN_TOKEN have been set. These keys have been set on my github repository but not yet on the official github repository. I plan to ask Wes soon to set these secrets on the official github repository.

Reply:
I googled this for a bit and unfortunately, GitHub does not appear to support a job-level conditional to only run a job when a secret is set.

Reply:
That's unfortunate. Anyway, thanks for having looked this up.